assets/scss/nhsuk-replacement-gutenberg.scss

```diff
@@ -45,6 +45,7 @@
 @import 'components/breadcrumb/breadcrumb';
 @import 'components/button/button';
 @import 'components/card/card';
+@import 'nhsblocks/legacy/care-card/care-card';
```

Comment:
That is potentially in a wrong place ... should be added at the end of nhuk-replacement.scss file alltogether with other assets

Reply:
This is deleted, thanks.